README.md

```diff
@@ -1,113 +1,125 @@
+      "maxConsecutiveFailures":0,
+      "path":"/healthz",
+      "portIndex":0,
+      "protocol":"HTTP"
     }
   ],
-  "instances": 1,
-  "mem": 128.0,
-  "ports": [
-    0,
-    1,
-    2
-  ]
+  "env":{
+    "FRAMEWORK_NAME":"etcd",
+    "FRAMEWORK_FAILOVER_TIMEOUT_SECONDS":"300",
+    "WEBURI":"http://etcd.marathon.mesos:$PORT0/stats",
+    "CLUSTER_SIZE":"3",
+    "MESOS_MASTER":"zk://localhost:2181/mesos",
+    "ZK_PERSIST":"zk://localhost:2181/etcd-mesos",
+    "VERBOSITY":"3",
+    "SINGLE_INSTANCE_PER_SLAVE":"true",
+    "AUTO_RESEED":"true",
+    "RESEED_TIMEOUT":"240",
+    "DISK_LIMIT":"1024",
+    "CPU_LIMIT":"0.2",
+    "MEM_LIMIT":"256"
+  }
 }
 ```
 
-## building
+## Build (optional)
 
-First, check out the tagged release for your version of Mesos.
+In order to build the code and, eventually, the Docker container image, one needs:
+* `make`, and
+* either `go` (1.8.1 is the recommended version) and [`govendor`](https://github.com/kardianos/govendor), or
+* Docker, in order to run the build in a container.
 
-For Mesos versions 23, 24, or 25, check out `v0.1.0-alpha-target-23-24-25`
+If all you want is to build the code, you have at least two ways to do it.
 
-For Mesos versions 22 and below (the farther below, the less the chances of compatibility), check out `v0.1.0-alpha-target-22`
-
-Next, built it!
-
-```
+Use `go build`:
+```sh
 make
 ```
 
-The important binaries (`etcd-mesos-scheduler`, `etcd-mesos-proxy`, `etcd-mesos-executor`) are now present in the bin subdirectory.
-
-A typical production invocation will look something like this:
-```
-/path/to/etcd-mesos-scheduler \
-    -log_dir=/var/log/etcd-mesos \
-    -master=zk://zk1:2181,zk2:2181,zk3:2181/mesos \
-    -framework-name=etcd \
-    -cluster-size=5 \
-    -executor-bin=/path/to/etcd-mesos-executor \
-    -etcd-bin=/path/to/etcd \
-    -etcdctl-bin=/path/to/etcdctl \
-    -zk-framework-persist=zk://zk1:2181,zk2:2181,zk3:2181/etcd-mesos
+or use Docker:
+```sh
+make docker_build
 ```
 
-If you'd like to build a new docker container, change the `DOCKER_ORG` and `VERSION` in the Makefile, and then run:
-```
+All necessary binaries are now available in the `bin` subdirectory.
+
+### Docker specifics
+
+If you'd like to build a new Docker container image:
+```sh
 make docker
 ```
 
-## service discovery
-Options for finding your etcd nodes on mesos:
+Optionally, set `DOCKER_ORG` and `VERSION`:
+```
+make docker DOCKER_ORG=quay.io/myorg VERSION=myver
+```
 
-* Run the included proxy binary locally on systems that use etcd.  It retrieves the etcd configuration from mesos and starts an etcd proxy node.  Note that this it not a good idea on clusters with lots of tasks running, as the master will iterate through each task and spit out a fairly large chunk of JSON, so this approach should be avoided in favor of mesos-dns on larger clusters.
+Optionally, push the container image:
+```sh
+ make docker DOCKER_ORG=quay.io/myorg VERSION=myver DOCKER_PUSH_ENABLED=1
```

Comment:
We are hosting in Dockerhub, it seems. I don't know who is able to push, though. Personally, I'm a fan of Quay.io because it's faster, can build images on its own and has container security scanning.

Will fix.